README.md

```diff
@@ -125,8 +125,12 @@ custom:
       - dataSource: # data source name
         type: # type name in schema (e.g. Query, Mutation, Subscription)
         field: getUserInfo
-        request: # request mapping template name
-        response: # response mapping template name
+        # pipeline resolvers use kind: PIPELINE, otherwise kind is not required
+        # kind: pipeline
```

Comment:
@markvp not required when UNIT, but still it would be accepted. I guess it does not hurt to specify it. 
Also, is ahs to be `PIPELINE` or `UNIT` (uppercase)

otherwise, [it will fail](https://github.com/sid88in/serverless-appsync-plugin/blob/master/src/index.js#L909)

Reply:
fixed